vf-react/src/js/components/layout/FilterModule.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
This Module displays, monitors, and alerts on changes to the filtering checkbox in the Modal popup.